### PR TITLE
added new validation pb114

### PR DESCRIPTION
**Pull request by JudahSchwartz**
add pb114, pb119 to the new validate infra.

### CODE REVIEW

File: demisto_sdk/commands/content_graph/objects/base_playbook.py [hunk 1]
```diff
@@ -175,6 +175,8 @@ def to_raw_dict(self) -> Dict:
 class BasePlaybook(ContentItem, content_type=ContentType.PLAYBOOK):  # type: ignore[call-arg]
     version: Optional[int] = 0
     tasks: Dict[str, TaskConfig] = Field([], exclude=True)
+    quiet: bool = Field(False)
+    inputs: List[Dict] = Field([])
```
Comment (YuvHayun) [inline demisto_sdk/commands/content_graph/objects/base_playbook.py:179]: Let's also add inputs to PB118

File: demisto_sdk/commands/content_graph/parsers/base_playbook.py [hunk 2]
```diff
@@ -81,6 +81,14 @@ def handle_playbook_task(self, task: Dict[str, Any], is_mandatory: bool) -> None
                 mandatorily=is_mandatory,
             )
 
+    @property
+    def inputs(self) -> Optional[List]:
+        return get_value(self.yml_data, 'inputs', [])
+
+    @property
+    def quiet(self) -> bool:
+        return get_value(self.yml_data, 'quiet', False)
```
Comment (YuvHayun) [inline demisto_sdk/commands/content_graph/parsers/base_playbook.py:90]: Add quiet and inputs to field_mapping, don't call them like this.

File: demisto_sdk/commands/validate/tests/PB_validators_test.py [hunk 3]
```diff
@@ -83,6 +86,34 @@ def test_is_valid_all_inputs_in_use(content_item, expected_result):
     )
 
 
+def test_playbook_quiet_mode_regular_playbook_pass():
+    playbook = create_playbook_object(["quiet"], [False])
+    assert PlaybookQuietModeValidator().is_valid([playbook]) == []
+
+
+def test_indicator_pb_must_be_quiet():
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/tests/PB_validators_test.py:94]: Add given-when-then.

File: demisto_sdk/commands/validate/validators/PB_validators/PB114_playbook_quiet_mode.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    error_code = "PB114"
+    description = "Validates that playbooks for indicator types will be on quiet mode."
+    rationale = (
+        "Playbooks for indicators will likely be executing on thousands of indicators "
+        "so they need to be on quiet mode."
+    )
+    error_message = (
+        "Playbooks with a playbookInputQuery for indicators should be on quiet mode."
+    )
+    fix_message = "This playbooks quiet mode was set to true"
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/validators/PB_validators/PB114_playbook_quiet_mode.py:25]:
```suggestion
    fix_message = "This playbooks quiet mode was set to true."
```

**Review by YuvHayun — APPROVED** [review]
Nice!
Minor comments.

File: demisto_sdk/commands/validate/tests/PB_validators_test.py [hunk 5]
```diff
@@ -83,6 +89,80 @@ def test_is_valid_all_inputs_in_use(content_item, expected_result):
...
 
 
+def test_using_input_not_provided():
+    """
+    Given:
+        inputs 2 and
+    When:
+        using input 3
+    Then:
+        Will fail on input 3 (and other defaults from test config)
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/tests/PB_validators_test.py:99]: Can you make it a little more clear?

File: .changelog/4348.yml [hunk 6]
```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Moved PB114, PB119 to the new validate infrastructure.
+  type: feature
```
Comment (JudahSchwartz) [inline .changelog/4348.yml:3]:
```suggestion
  type: internal
```

File: demisto_sdk/commands/validate/tests/PB_validators_test.py [hunk 7]
```diff
@@ -83,6 +89,80 @@ def test_is_valid_all_inputs_in_use(content_item, expected_result):
...
+    playbook = create_playbook_object(["quiet"], [False])
+    assert PlaybookQuietModeValidator().is_valid([playbook]) == []
+
+
+def test_indicator_pb_must_be_quiet():
+    """
+    Given:
+        A pb with queryEntity indicators
+    When:
+        in quiet mode
```
Comment (JudahSchwartz) [inline demisto_sdk/commands/validate/tests/PB_validators_test.py:139]:
```suggestion
        not in quiet mode
```

**Pull request merged**